sktime/utils/datetime.py

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env python3 -u
 # -*- coding: utf-8 -*-
+"""
+Times series functions related utilities.
+"""
 
-__author__ = ["Markus Löning"]
+__author__ = ["Markus Löning", "Yang Zhang"]
```

Comment:
this should be GitHub IDs, i.e., `mloning` etc

Reply:
I have added the name to align with the author name format in other files (the original authors seem to put their names instead of the Github IDs). I assume both the author names and GitHub IDs should be OK? Let me know if not, I can surely change it to the Github IDs.

Reply:
> Yes let's use GitHub IDs here, that will be more useful, we previously used the author names, but using the GitHub IDs will make it easier to link to the user profiles in the [estimator overview](https://www.sktime.org/en/stable/estimator_overview.html)

OK. I see. Will change to Github IDs then.